react-example/server/server.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable consistent-return */
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const express = require('express');
+const { exec } = require('child_process');
+const cors = require('cors');
+
+const app = express();
+const port = 3000;
+
+app.use(cors({ origin: '*' }));
+app.use(express.json());
+
+app.post('/generate', (req, res) => {
+  const { jwt_secret, userId, email, exp_minutes } = req.body;
```

Comment:
Hi, I have a question about `jwt_secret`. Why is `jwt_secret` passed from the client? Shouldn't it be hidden on the server? Thanks!

Reply:
This is just an example for local generation of JWTs for testing.